.github/ISSUE_TEMPLATE/icon_removal.yml

```
@@ -0,0 +1,36 @@
+name: Icon removal
+description: Report an icon for removal
+title: "Remove: "
+labels: [removal request, breaking change]
+
+body:
+  - type: markdown
+    attributes:
+      value: >-
+        Before opening a new issue, make sure it isn't covered by an existing issue.
+        Please search for [issues with the `removal request` label][removal-request-issues]
+        (including closed issues) before you continue. If you find one for the brand
+        you're reporting then leave a comment on it or add a reaction.
+
+
+        [removal-request-issues]: https://github.com/simple-icons/simple-icons/issues?q=is%3Aissue+label%3A%22removal+request%22+is%3Aopen+
+
+  - type: input
+    attributes:
+      label: Brand Name
+      placeholder: "Example: Simple Icons"
+    validations:
+      required: true
+
+  - type: textarea
+    attributes:
+      label: Reason for Removal
+      description: |
+        Include all relevant links, for example:
+        - The website announcing the company was closed
+        - A blogpost announcing the brand has been discontinued
```

I feel like providing specific examples for how a brand might have announced it's discontinued here isn't very productive. Also, a discontinued brand is not the only reason for a removal request (see e.g. [the disclaimer](https://github.com/simple-icons/simple-icons/blob/develop/DISCLAIMER.md#removal-of-brands)).

```suggestion
        - If the brand no longer exists, include a link supporting that claim.
        - If the brand should be removed for another reason, explain why.
```